src/component-library/utils/chain.ts

```diff
@@ -0,0 +1,10 @@
+// Calls all functions in the order they were chained with the same arguments.
```

Comment:
Might have been a misunderstanding earlier, but I'm fine importing this from 'react-stately` instead of copying it into our own utility.

Reply:
Sorry. I understood the opposite. I have applied the change.